docs/9.x/config.md

```diff
@@ -61,6 +61,9 @@ spec:
     # Enables Kubernetes high availability mode. When HA mode is enabled, Kubernetes
     # control plane components run on all master nodes.
     highAvailability: true
+    # The flannel backend is automatically selected based on the cloud provider,
+    # but the selected backend can be overridden if specified in the cluster configuration.
```

Comment:
For docs, I would also consider including what the valid values are. We're not good at doing this elsewhere, but it's one of those little things that when looking at the docs someone can figure out what they can/cannot add.